executor/executor_required_rows_test.go

```diff
@@ -575,7 +575,7 @@ func (s *testExecSuite) TestProjectionParallelRequiredRows(c *C) {
 
 			// wait projectionInputFetcher blocked on fetching data
 			// from child in the background.
-			time.Sleep(time.Millisecond * 5)
+			time.Sleep(time.Millisecond * 25)
```

Comment:
Do we have a deterministic approach to do the synchronization?

Reply:
No.. we can't trace states of these background goroutines now 😢..
And it's ugly to introduce this feature just for this unit test..